### PR TITLE
Support the "Relax IEEE compliance" settings

### LINKED ISSUE #134: The "Relax IEEE compliance" build configuration setting is not supported

### LINKED ISSUE #152: The "Relax IEEE compliance" build configuration setting is not supported

### PR DESCRIPTION
**Pull request by McMartin**
Resolves #134.
Resolves #152.

@MartyLake: please review, thanks!

### CODE REVIEW

**Comment by MartyLake** [comment]
Is the cmake setting missing for OSX / Linux? I only saw the windows one.

File: cmake/Reprojucer.cmake [hunk 1]
```diff
@@ -579,6 +579,7 @@ function(jucer_export_target_configuration exporter NAME_TAG configuration_name)
       "OSX_DEPLOYMENT_TARGET"
       "OSX_ARCHITECTURE"
       "CXX_LANGUAGE_STANDARD"
+      "RELAX_IEEE_COMPLIANCE"
```
Comment (McMartin) [inline cmake/Reprojucer.cmake]: @MartyLake: this is for macOS (see line 576)

File: cmake/Reprojucer.cmake [hunk 2]
```diff
@@ -684,6 +686,12 @@ function(jucer_export_target_configuration exporter NAME_TAG configuration_name)
           )
         endif()
 
+      elseif(tag STREQUAL "RELAX_IEEE_COMPLIANCE" AND exporter STREQUAL "Xcode (MacOSX)")
```
Comment (McMartin) [inline cmake/Reprojucer.cmake]: @MartyLake: this is for macOS as well

**Comment by McMartin** [comment]
@MartyLake: the Projucer doesn't have a "Relax IEEE compliance" setting on Linux, only on macOS and Windows.

**Comment by MartyLake** [comment]
LGTM then !


---
Matthieu TALBOT
neogaldr@gmail.com

+33 629 22 15 70
fr.linkedin.com/in/matthieutalbot
http://www.soundcloud.com/Marty-Lake
http://lifehacker.com/5028808/how-eom-makes-your-email-more-efficient

**Pull request merged**